### ISSUE TITLE
"oscap xccdf resolve" discards / resets / removes the value of @update URI attribute from <xccdf:version> element

### ISSUE THREAD

**Issue by iankko** [OpenSCAP/openscap#366]
From the [XCCDF v1.2 specification](http://csrc.nist.gov/publications/nistir/ir7275-rev4/NISTIR-7275r4.pdf) (Page #24):
```
version    (element)    string   1    Version number of the benchmark, with an OPTIONAL
                                      @time timestamp attribute (when the version was completed)
                                      and an OPTIONAL @update URI  attribute
                                      (where updates may be obtained).
```

Providing an ```<xccdf:version>``` element in the form of e.g.:
```
<version update="https://github.com/OpenSCAP/scap-security-guide/releases/latest">0.1.28</version>
```
(IOW having the optional @update attribute set) and resolving such a benchmark via oscap:

```oscap xccdf resolve -o /tmp/resolved.xml /tmp/unresolved```

leads to value of ```@update``` attribute of ```<xccdf:version>``` to be removed / cleared.

How reproducible:
Always

Steps to reproduce:
(The following steps assumes this has been already done:

git clone scap-security-guide
cd scap-security-guide/RHEL/6 && make content
)

```
$ grep '<xccdf:version' xccdf-unlinked-unresolved.xml 
  <xccdf:version xmlns:xccdf="http://checklists.nist.gov/xccdf/1.1">0.1.28</xccdf:version>
$ oscap xccdf resolve -o /tmp/resolved.xml xccdf-unlinked-unresolved.xml
$ echo $?
0
$ grep '<version' /tmp/resolved.xml 
  <version>0.1.28</version>
```
Current result:
@update attribute value in ```<xccdf:version>``` element is reset during XCCDF resolving.

Expected result:
@update attribute value in ```<xccdf:version>``` is kept after XCCDF resolving.

Additional information:
See attached ```xccdf-unlinked-unresolved.xml``` for an example of small XCCDF benchmark having @update attribute set.

**Comment by iankko** [comment]
Promised example

[xccdf-unlinked-unresolved.xml.zip](https://github.com/OpenSCAP/openscap/files/193499/xccdf-unlinked-unresolved.xml.zip)

**Comment by iankko** [comment]
To my knowledge, this (already merged) patch is a fix for this issue:
&nbsp; &nbsp; [1] https://github.com/OpenSCAP/openscap/commit/1f05cc104a34f49e34e27f616280837e5c68388d

I have filed this bug, because I need a ticket reference for SSG pull request (also the information how to reproduce might be found being usable some day).

**Comment by mpreisler** [comment]
Hi,
as you said this was fixed and the fix will be released with `1.0.11` and `1.2.9`. If you can reproduce this issue with git openscap please reopen.
